.travis.yml

```diff
@@ -7,21 +7,43 @@ cache:
 
 language: php
 
+## Cache composer downloads.
+cache:
+  directories:
+    # Cache directory for older Composer versions.
+    - $HOME/.composer/cache/files
+    # Cache directory for more recent Composer versions.
+    - $HOME/.cache/composer/files
+
 php:
   - 5.5
   - 5.6
   - 7.0
-  - 7.1
```

Comment:
The 7.1 builds are the ones with coveralls and defined in the matrix lower down.
Similarly, the 5.3 and 5.4 builds (partially) need `dist: precise` and some of them need the coveralls env variable, which is why those are also in the matrix `include` part.